R/examples.R

```diff
@@ -0,0 +1,23 @@
+valid_examples <- c(
+  "circularLinearWithBrush", "circularVisualEncoding", "multiTrackApp", "sarsCov2", "staticCircularBar",
+  "structuralVariant", "textAnnotation"
+)
+
+package_name <- "shiny.gosling"
+
+#' Runs a shiny.gosling example
+#' @param example A character indicating a valid example.
+#'
+#' @export
```

Comment:
Please add a runnable example. This is mandatory for bioc. At least 80% of man page func docs that are exported should have runnable examples. Right now there is just enough to reach 80%. One more exported func without runnable example will produce error on bioc check.